examples/translation_example.py

```diff
@@ -0,0 +1,406 @@
+        ddf["text"] = ddf["text"].apply(self.custom_tokenize)
+        ddf["doc_id"] = range(1, len(ddf) + 1)
+        ddf = ddf.explode("text", ignore_index=True)
+        ddf = ddf.reset_index(drop=False)
+        ddf = cudf.DataFrame.from_pandas(ddf)
+        return ddf
+
+    def concat_strings(self, series):
+        return "".join(series)
+
+    def combine_text(self, ddf):
+        engligh_stop_flag = ddf["text"].str.endswith(".")
+        hindi_stop_flag = ddf["translation"].str.endswith("|")
+        ddf["translation"][~engligh_stop_flag & hindi_stop_flag] = ddf[
+            "translation"
+        ].str.rstrip("|")
+        ddf["translation"] = ddf["translation"].str.strip()
+        return ddf
+
+    def grouping(self, ddf):
+        ddf = ddf.to_pandas()
+        agg_funcs = {
+            "translation": self.concat_strings,
+            "text": self.concat_strings,
+        }
+        other_columns = {
+            col: "first"
+            for col in ddf.columns
+            if col not in agg_funcs and col != "doc_id"
+        }
+
+        agg_funcs.update(other_columns)
+        ddf = ddf.groupby("doc_id").agg(agg_funcs).reset_index()
+        ddf = cudf.DataFrame.from_pandas(ddf)
+        return ddf
+
+    def atleast_letter(self, df):
+        df = df.to_pandas()
+        df["isalpha"] = df["indic_proc_text"].apply(self.has_alphabet_characters)
+        df = cudf.DataFrame(df)
+        return df
+
+    def _run_classifier(self, dataset: DocumentDataset):
+        st = time.time()
+        ddf = dataset.df
+        ddf_meta = ddf._meta.copy()
+        ddf_meta["doc_id"] = ""
+        ddf_meta["index"] = ""
+        # Applying process_input_text for following :
+        # 1. nltk tokenization to break doc into sentences
+        # 2. craeting a row w.r.t each sentence.
+        # 3. Process sentences strip symbols from start and end
+        ddf = ddf.map_partitions(self.process_input_text, enforce_metadata=False)
+
+        ddf["word_count"] = ddf["text"].str.split().list.len()
+        ddf["word_count"] = ddf["word_count"].astype("int64")
+        ddf_true = ddf[(ddf["word_count"] <= self.translation_config.max_words_per_sen)]
+
+        ddf_metaa = ddf_true._meta.copy()
+        ddf_metaa["isalpha"] = False
+        # To filter for atleast one unicode letter in text
+        has_letter = ddf_true.map_partitions(self.atleast_letter, meta=ddf_metaa)
+        ddf_trans = ddf_true[has_letter["isalpha"]]
+        ddf = ddf_trans.drop(columns="word_count")
+
+        ## ddf false operations
+        ddf_false = ddf_true[~has_letter["isalpha"]]
+        ddf_false = ddf_false.drop(columns="word_count")
+        ddf_false_meta = ddf_false._meta.copy()
+        ddf_false_meta["translation"] = ""
+        ddf_false["translation"] = ddf_false["text"]
+
+        # Applying preprocess_df for Indic preprocessing
+        ddf["text"] = ddf["text"].astype("str")
+        ddf_meta = ddf._meta.copy()
+        ddf_meta["indic_proc_text"] = ""
```

Comment:
Please test code before pushing. This does not work. I am working on a fix from my best understanding.